osu.Game/Rulesets/Mods/ModNoScope.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Localisation;
+using osu.Game.Configuration;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Overlays.Settings;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Scoring;
+using osu.Game.Screens.Play;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public abstract class ModNoScope : Mod, IApplicableToScoreProcessor, IApplicableToPlayer
+    {
+        public override string Name => "No Scope";
+        public override string Acronym => "NS";
+        public override ModType Type => ModType.Fun;
+        public override IconUsage? Icon => FontAwesome.Solid.EyeSlash;
+        public override double ScoreMultiplier => 1;
+
+        /// <summary>
+        /// Slightly higher than the cutoff for <see cref="Drawable.IsPresent"/>.
+        /// </summary>
+        protected const float MIN_ALPHA = 0.0002f;
+
+        protected const float TRANSITION_DURATION = 100;
+
+        protected BindableNumber<int> CurrentCombo;
+
+        protected IBindable<bool> IsBreakTime;
+
+        protected float ComboBasedAlpha;
+
+        [SettingSource(
+            "Hidden at combo",
+            "The combo count at which the cursor becomes completely hidden",
```

Comment:
Weird for this to mention the "cursor" in the base implementation, while it's actually going to affect the catcher in catch.

Reply:
Made separated descriptions for each mod, but this leads to duplication. Maybe it's better to change "cursor" -> "player" in the base implementation?

Reply:
I don't mind duplications. Chances are the ruleset implementation may diverge in more ways than that anyhow.